.travis.yml

```diff
@@ -65,8 +65,7 @@ before_script:
   - sleep 3 # give xvfb some time to start
 
 script:
-    - if [ "$DISTRIB" == "ubuntu 14" ]; then python setup.py nosetests; fi
-    - if [ "$DISTRIB" == "conda" ]; then nosetests; fi
+    - python setup.py test
```

Comment:
Is there an advantage compared to using something like pytest sphinx_gallery? I think we dropped this in joblib because python setup.py test is just not that common in my experience. Happy to be proven wrong.